evergreen/system_perf/5.0/genny_tasks.yml

```diff
@@ -43,7 +43,6 @@ buildvariants:
   - name: multi_updates_multi_updates
 - name: perf-3-node-replSet.arm.aws.2023-11
   tasks:
-  - name: device_monitoring
```

Comment:
Why are we removing device_monitoring? I don't see this as part of the task.

Reply:
That was removed form the PrivateWorkloads repo as part of [PERF-5941](https://jira.mongodb.org/browse/PERF-5941). 
Task lists in genny are now getting auto updated based on that removal.